context.go

```diff
@@ -179,6 +179,10 @@ func (c *Ctx) resolveProjectRoot(path string) (string, error) {
 //
 // The second returned string indicates which GOPATH value was used.
 func (c *Ctx) SplitAbsoluteProjectRoot(path string) (string, error) {
+	if path == filepath.Join(c.GOPATH, "src") {
```

Comment:
Wouldn't comparing the `len`s of `path` and `$GOPATH/src` cause this error to be triggered if they happen to be of the same length. (e.g. if $GOPATH=`/Users/user/Code/go/src` and the `dep init` is run in `/Users/user/Code/marvel`...)?

I originally compared the `len`s and then switched to comparing the strings.